Makefile

```diff
@@ -960,12 +960,19 @@ shell: $(GOBUILDER)
 	$(QUIET)DOCKER_GO_ARGS=-t ; $(DOCKER_GO) bash $(GOTREE) $(GOMODULE)
 
 #
-# Utility targets in support of our Dockerized build infrastrucutre
+# Linuxkit
 #
+.PHONY: linuxkit
+linuxkit: $(LINUXKIT)
```

Comment:
I don't see `linuxkit` as a dependency for any other target, so I think it's better to remove it. Also, targets relying on linuxkit can define `$(LINUXKIT)` as dependency...

Reply:
@rene this is a phony target for testing the buld of LK

Reply:
I agree. It is convenient to be able to run `make linuxkit`. That having been said, maybe `linuxkit` should be the dependency of other targets rather than `$(LINUXKIT)`? It leads to the same thing, but is a little easier to understand? Doesn't matter all that much, I guess.

Reply:
@deitch NO!!!! read about dependencies on PHONY targets :) they will be ALWAYS rebuilt

Reply:
@deitch for the same reason I'm going to remove 'current' target